config/database.yml

```diff
@@ -1,25 +1,21 @@
-# SQLite version 3.x
-#   gem install sqlite3
+# MySQL version 2.x
+#   gem install mysql2
 #
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
+#   Ensure the MySQL 2 gem is defined in your Gemfile
+#   gem 'mysql2'
 #
-default: &default
-  adapter: sqlite3
-  pool: 5
-  timeout: 5000
 
-development:
-  <<: *default
-  database: data/db/development.sqlite3
+default: &default
+  url: <%= ENV['DATABASE_URL'] %>/predeploychecker
```

Comment:
Couple things here, and sorry for the thrash:

1. the url should only be used for production, test and development should be using username/password/database/adapter 
2. the default should still be configuring variables like timeout, pool size, and adapter

Example
```yml
default: &default
  adapter: mysql2
  pool: 5
  username: <%= ENV['DATABASE_USER'] || rr-dev %>
  password: <%= ENV['DATABASE_PASS'] || dev %>
  host: 127.0.0.1

test:
  <<: *default
  database: predeploychecker_test

development:
  <<: *default
  database: predeploychecker_dev

production:
  url: <%= ENV['DATABASE_URL'] %>
```

Reply:
No worries. I see what you are saying. Thanks for the help!